lib/material/measing.flow

```diff
@@ -698,7 +700,7 @@ MZoomAnimation(
 		],
 		MMEasingAnimation(
 			const(0.3),
-			const(easingFastOutSlowIn),
+			const(CubicBezierEasing(0.42, 0.0, 0.58, 1.0)),
```

Comment:
Strange this constants in a component.
 @NikitaFil maybe you know better.

Reply:
It isn't a standard curve, but no objections about custom curve usage